Doc/library/mmap.rst

```diff
@@ -81,7 +81,11 @@ To map anonymous memory, -1 should be passed as the fileno along with the length
    private copy-on-write mapping, so changes to the contents of the mmap
    object will be private to this process, and :const:`MAP_SHARED` creates a
    mapping that's shared with all other processes mapping the same areas of
-   the file.  The default value is :const:`MAP_SHARED`.
+   the file. :const:`MAP_ALIGNED_SUPER` (FreeBSD only) allows to map to
+   pages of 2MB (x86) or so, it is enabled by default, at boot time,
+   in most of architectures and exists since 2008.
+   :ref:`MAP_* constants <mmap-constants>`.
```

Comment:
"MAP_* constants" is not really a sentence. Maybe move it after the default, and write something like:

```
The default value is :const:`MAP_SHARED`. 
See :ref:`MAP_* constants <mmap-constants>`.
```